ads/google/correlator.js

```diff
@@ -24,6 +24,8 @@ export function makeCorrelator(clientId, pageViewId) {
   if (clientId) {
     return pageViewIdNumeric + (clientId.replace(/\D/g, '') % 1e6) * 1e6;
   } else {
-    return pageViewIdNumeric;
+    // In this case, pageViewIdNumeric is only 4 digits => too low entropy
+    // to be useful as a page correlator.  So synthesize one from scratch.
+    return Math.floor(Math.pow(2, 52) * Math.random());
```

Comment:
I read from somewhere Math.random gives about 32bit entropy. Maybe you get the 52 from somewhere else. 

Whatever number we decided finally, can you just hard code it instead of doing a Math.pow() ?